### PR TITLE
ncmpcpp: add page

### PR DESCRIPTION
**Pull request by mstruebing**
----

As `ncmpcpp` is a widely used music player I think it would be nice to add this to the pages.

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/ncmpcpp.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# ncmpcpp
+
+> Ncmpcpp is a command line music player.
```
Comment (sbrl) [inline pages/common/ncmpcpp.md]: Perhaps we could reword this such that the command name isn't mentioned in its description? Here's the description of the command from the apt package: https://hastebin.com/tamososaya

So perhaps we could do something like `A command line music player client for the Music Player Daemon?`?

File: pages/common/ncmpcpp.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Connect to a music player daemon on a given host and port:
+
+`ncmpcpp --host {{ip}} --port {{port}}`
+
+- Print the current song to stdout:
+
+`ncmpcpp --current-song`
+
+- Use specific config:
```
Comment (sbrl) [inline pages/common/ncmpcpp.md]: Perhaps `Use a specified configuration file` would read better?

File: pages/common/ncmpcpp.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`ncmpcpp --config {{file}}`
+
+- Use different key bindings:
```
Comment (sbrl) [inline pages/common/ncmpcpp.md]: Perhaps `Use a different set of key bindings from a file` would read better? I'm not sure here - that doesn't sound quite right either.

**Comment by mstruebing** [comment]
@sbrl adjusted.

**Comment by sbrl** [comment]
Oops! Looks like I made a typo in that comment :sweat_smile:

File: pages/common/ncmpcpp.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# ncmpcpp
+
+> A command line music player client for the Music Player Daemon.
+
+- Connect to a music player daemon on a given host and port:
+
+`ncmpcpp --host {{ip}} --port {{port}}`
+
+- Print the current song to stdout:
```
Comment (agnivade) [inline pages/common/ncmpcpp.md]: What does it mean to print the song to stdout ? Does it show the lyrics or something else ?

Comment (mstruebing) [reply pages/common/ncmpcpp.md]: It shows the artist name and song name, I will clarify it when I'm at work.

Comment (agnivade) [reply pages/common/ncmpcpp.md]: Ok, maybe better to say "Display metadata about the current song to console". Try to avoid tech jargon like "stdout"

**Comment by sbrl** [comment]
Thanks, @mstruebing!

**Pull request merged**